### PR TITLE
Font size and rte link

### PR DESCRIPTION
**Pull request by simonorchanian**

### CODE REVIEW

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java [hunk 1]
```diff
@@ -31,36 +31,36 @@ public String clickOnCreateLink() throws InterruptedException  {
...
+					      Thread.sleep(2000);
+					      waitForEmailCampaignsPage.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector(".container-main"))); 		      
+					      
+					      //Verify that the Pause button shows up
+						  WebElement verifyDeleteButton = driver.findElement(deleteButtonLink);
+							
+							  if(verifyDeleteButton.isDisplayed()) 
+							  	{
+							       String getDeleteButton = verifyDeleteButton.getText();	
+								   return getDeleteButton;
```
Comment (Morgantheplant) [inline Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: getDeleteButton sounds like a method name not a string

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java [hunk 2]
```diff
@@ -31,36 +31,36 @@ public String clickOnCreateLink() throws InterruptedException  {
...
+					            }		  				   
+						else	             	   
 					 System.out.println("Element not found");
```
Comment (Morgantheplant) [inline Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: Seems like the error could be more descriptive here

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java [hunk 3]
```diff
@@ -71,20 +71,18 @@ public boolean clickNewEmailCampaignButton() throws InterruptedException {
...
+			       if (newEmailCampaignButton.isDisplayed())
+			       {   
 					  newEmailCampaignButton.click(); 
-				
-					  
-				     // Wait for the email campaign link to be visible
-			    	 Thread.sleep(2000);
+		  
+					  // Wait for the email campaign link to be visible
+					  Thread.sleep(2000);
```
Comment (Morgantheplant) [inline Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: is 2000 an arbitrary value?

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java [hunk 4]
```diff
@@ -127,38 +127,38 @@ public String composeEmailFormFillout() throws InterruptedException {
 			    if(composeEmailScheduleLink.isDisplayed()) 
 			    {
 				  composeEmailScheduleLink.click();
-				  Thread.sleep(2000);
+				  Thread.sleep(3000);
```
Comment (Morgantheplant) [inline Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: why 3000 here?

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java [hunk 5]
```diff
@@ -176,9 +176,17 @@ public boolean verifyRelativeEmailCampaignOnUser() throws InterruptedException {
...
-			 return true; 
-			}
-
+			 
+			 WebElement getCampaignName = driver.findElement(By.cssSelector("tr:nth-child(2) > td:nth-child(1) > div > a"));
+			 String userCampaignName = getCampaignName.getText();
+			 
+			 	if (userCampaignName.contains("Hearsay Systems Automation"))
+				  { 
+			 		 return true;
```
Comment (Morgantheplant) [inline Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: could just be 
```
return userCampaignName.contains("Hearsay Systems Automation")
```

**Comment by Morgantheplant** [comment]
lgtm couple of minor nits/q's

**Comment by simonorchanian** [comment]
Thanks Morgan ;   I'll have a look / refactor

**Comment by gdtrice** [comment]
There's a lot of sleep calls, which could lead to flaky tests. Are there any better ways to pause before executing further?

**Comment by dakaugu** [comment]
squash commits

**Comment by dakaugu** [comment]
seems like the whole file suffers from poor indentation. can we run `⌘ + ⌥  + L` if using jet brains ide

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java (hunk 2 above)
Comment (simonorchanian) [reply Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: Done.

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java (hunk 3 above)
Comment (simonorchanian) [reply Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: It is;   It's not good practice to use sleeps but I find them necessary sometimes.

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java (hunk 4 above)
Comment (simonorchanian) [reply Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: Changed to 2000;   sometimes there's more lag so I bump it up a bit.....

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java (hunk 5 above)
Comment (simonorchanian) [reply Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: Done.

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java (hunk 1 above)
Comment (simonorchanian) [reply Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: Modified it.

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java (hunk 3 above)
Comment (Morgantheplant) [reply Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: I am not really familiar with WebDriver, but would a FluentWait not work here?
Quick search looks like it gives some useful options for polling and timeout
```
// Waiting 30 seconds for an element to be present on the page, checking
// for its presence once every 5 seconds.
Wait<WebDriver> wait = new FluentWait<WebDriver>(driver)
    .withTimeout(30, SECONDS)
    .pollingEvery(5, SECONDS)
    .ignoring(NoSuchElementException.class);
```

**Comment by simonorchanian** [comment]
@gdtrice , @Morgantheplant :  I've refactored the sleeps (except for a couple) to explicit waits ;  
Tests works pretty good too.  Thanks.

File: Code/src/PageObjects/AdminEmailRelativeCampaigns.java (hunk 3 above)
Comment (simonorchanian) [reply Code/src/PageObjects/AdminEmailRelativeCampaigns.java]: Yep, I've already modified the code to use explicit waits.

File: Code/src/PageObjects/CheckFontSizeAndRTELink.java [hunk 6]
```diff
@@ -0,0 +1,115 @@
...
+
+	public void method2() { 
+		// TODO Auto-generated method stub
+	}	  
+} 
+
+
+
+
+
```
Comment (dakaugu) [inline Code/src/PageObjects/CheckFontSizeAndRTELink.java]: blank lines

File: Code/src/Tests/CheckFontSizeAndRTELinkTest.java [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+		// verify the link, page, tiles, and filter link
+		System.out.println("Verifying composing and scheduling email");
+		Assert.assertTrue(checkFontSizeAndRTELink.verifyFontSizeAndRTELinks());
+
+
+		driver.close();
+	}
+
+
+
```
Comment (dakaugu) [inline Code/src/Tests/CheckFontSizeAndRTELinkTest.java]: remove empty lines

File: Code/src/Tests/CheckFontSizeAndRTELinkTest.java [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+	@AfterTest
+	public void tearDown() {
+		driver.quit();
+	}
+
+}
+
+
```
Comment (dakaugu) [inline Code/src/Tests/CheckFontSizeAndRTELinkTest.java]: emply

**Comment by dakaugu** [comment]
some of these files in this pr could still have better indentations

**Comment by dakaugu** [comment]
squash commits

**Comment by simonorchanian** [comment]
@dakaugu  or  @Morgantheplant :  Is this PR ready to be merged in?  Thank you.

**Pull request merged**